content/docs/command-reference/commit.md

```diff
@@ -46,6 +41,19 @@ scenarios are further detailed below.
   reproduce the whole pipeline. If you're sure no pipeline results would change,
   use `dvc commit` to force update the `dvc.lock` or `.dvc` files and cache.
 
+- Sometimes we have executed a stage (either by writing `dvc.yaml` manually and
+  using `dvc repro`, or with `dvc run`), and later notice that output(s) which
+  already exist in workspace are missing in `dvc.yaml`. We can
```

Comment:
Good but I wouldn't start with "Sometimes" again same as the previous bullet (minor), and please apply the same changes you do to the paragraph in content/docs/user-guide/how-to/add-output-to-stage.md per the recent comments on files vs. outputs I left there. Thanks

Reply:
and please apply the same changes you do to the paragraph in content/docs/user-guide/how-to/add-output-to-stage.md

Reply:
Still pending... Will comment again.